buildSrc/src/main/kotlin/version-file-conventions.gradle.kts

```diff
@@ -0,0 +1,17 @@
+import org.gradle.api.tasks.bundling.*
+
+/* `kotlinx-coroutines-debug` configures `VersionFile` on its own when the corresponding task is created. */
+val invalidModules = listOf("kotlinx-coroutines-debug")
+
+configure(subprojects.filter {
```

Comment:
Thank you, the Archangel. The wisdom of this advice is indisputable. Yet our land is too full of sin to be ready for such a revelation. The project is ripe with the filthy touch of subproject iteration, and making one place holy will but incite more chaos. The whole city must be burned to crisps before it rises as a temple (if you catch my meaning).

Though if you have quick access to some specific links to the required reading, sure, please share them. Maybe I'm wrong, and this could conveniently be done one iteration over subprojects at a time, then there's no reason not to do that.

Reply:
Hahahahaha.

I did not look into details yet, I may not have time today, but maybe if I'm bored this weekend, we never know!
But indeed, feel free to ignore my comment for now, it was mainly informative and we probably should nuke all of these cross-configuration at once.